### DISCUSSION TITLE
Setting up for compilation on Windows

### DISCUSSION THREAD

**Discussion by DesertDuplos** [rgriebl/brickstore#446] [Q&A]
I am wanting to try some things out within BrickStore.  I am a SW Engineer, although it has been a while since I have coded (I got demoted into managing the team).  I have also never had experience with setting up the build since I never have played with that stuff at home and it was managed by others at work.

BrickStore seemed like a great opportunity to try coding again, and have some side benefits for the BrickLink store that I run.  I didn't realize how "new" it would seem.

I have MS Visual Studio, and I have pulled down the git repository for BrickStore.  I have also downloaded Qt.  I am at a loss with how to connect this together so that the BrickStore project is recognizing and building with Qt.  If there is an easy "gotcha" answer, great.  If I really need to dig into certain tutorials or training, great as well.

Thanks!
David
Desert Duplos

**Comment by rgriebl** [comment]
Just to get started:
 start  QtCreator, open the project file `brickstore.pro` and click the "Play" button (or press `Ctrl-R`)

If you want to work on the codebase, you could use Visual Studio (*), but I personally do everything in QtCreator nowadays - it's just the better IDE for my taste.

(*) at least use the Qt extensionl https://doc.qt.io/qtvstools/index.html to make your life easier.
